OpenChange/RTFHandler.m

```diff
@@ -688,21 +683,11 @@ - (void) parsePicture
               binary_size = [s intValue];
               ADVANCE_N(binary_size);
             }
-          else if (len > 0)
-            {
-              ADVANCE_N(len);
```

Comment:
This was just wrong because `[self parseControlWord: &len]` is already moving forward so no need to advance